typescript-id-allocator/IdCompressor.ts

```diff
@@ -32,6 +32,7 @@ export class IdCompressor implements IIdCompressor, IIdCompressorCore {
 		return new IdCompressor(new WasmIdCompressor(localSessionId), localSessionId);
 	}
 
+	// TODO: A consumer passing an invalid or unknown SessionId will produce a JsError (at wasmCompressor.get_token()). Is this the behavior we want?
```

Comment:
I think it is, yes. We just probably want a good error message in that JsError (we have an issue for that).

Reply:
Can probably kill this todo

Reply:
My concern here was that this is an instance of panicking / exiting as a result of incorrect user input. The method name and signature imply that it will always either get or create you a token, but instead it fails when trying to create a token for an incorrect input (such as passing the wrong sessionID to normalize). Thoughts?

Reply:
so, it won't panic in Rust, so that leaves the state recoverable. if an app is worried about somehow passing in bad session ids (how would they even get one without intentionally casting?) they can wrap their call to this method in a try/catch.

Reply:
Good point. I was misremembering the expected external behavior of JsError. I thought that it caused an exit regardless.